docs/schema/github.md

```diff
@@ -174,7 +174,8 @@ Representation of a single Programming Language [language object](https://develo
 
 ## Dependency::PythonLibrary
 
-Representation of a Python library as listed in a [requirements.txt](https://pip.pypa.io/en/stable/user_guide/#requirements-files) file.
+Representation of a Python library as listed in a [requirements.txt](https://pip.pypa.io/en/stable/user_guide/#requirements-files)
+or [setup.cfg](https://setuptools.pypa.io/en/latest/userguide/declarative_config.html) file
```

Comment:
Probably worth mentioning that we include everything from all of `install_requires`, `setup_requires`, and `extras_require`, folks might only think about `install_requires` at first.